src/sinks/websocket/config.rs

```diff
@@ -28,12 +28,25 @@ pub struct WebSocketSinkConfig {
     #[configurable(derived)]
     pub encoding: EncodingConfig,
 
-    /// The interval, in seconds, between sending PINGs to the remote peer.
+    /// The interval, in seconds, between sending [Ping][ping]s to the remote peer.
+    ///
+    /// If this option is not configured, pings are not sent on an interval. This option must be configured to a value > 0
+    /// in order to enable pings.
```

Comment:
I wonder if it's worth making this a `NonZeroU64`?

Reply:
ill take a look at that. good suggestion. I think as it is, the value of zero is discarded silently, so that would be a great improvement.

Reply:
Yeah, just trying to leverage the wonderful type system :D